### PR TITLE
NJ YAML files updated.

### PR DESCRIPTION
**Pull request by brianembley**
*Issue #, if available:*

*Description of changes:*
Added details including tags and Tutorial info.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 95395fc1ab4904b6d127e6d936320b1c21e1d907
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=75070c62-e0de-4ad2-8207-226d1bd6c1b6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/NJ_Imagery.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+UpdateFrequency: None
+Tags:
+  - aerial imagery
+  - earth observation
+  - geospatial
+  - imaging
+  - mapping
+License:  The State of New Jersey provides the service "as is". The State makes no guarantee or warranty concerning the accuracy of information contained in the data. Also, the State makes no warranty, either expressed or implied, regarding the condition of the product or its fitness for any particular purpose. The burden for determining fitness for use lies entirely with the user. Although these data files and service have been processed successfully on computers at the State, no warranty is made by the State regarding the use of the data or service on any other system, nor does the fact of distribution constitute or imply such a warranty.
+Resources:
+  - Description: https://njgin.nj.gov/njgin/edata/imagery
```
Comment (jflasher) [inline datasets/NJ_Imagery.yaml:14]: Please change to a text description of the bucket contents. Can add link if needed, but should also be a brief description here.

File: datasets/NJ_Imagery.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    ARN: arn:aws:s3:::njogis-imagery
+    Region: us-west
```
Comment (jflasher) [inline datasets/NJ_Imagery.yaml:16]: I believe this should be us-west-2

File: datasets/NJ_Imagery.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Name: Digital Aerial Imagery Catalog
```
Comment (jflasher) [inline datasets/NJ_Imagery.yaml:1]: Please add something indication this is from NJ, something like below
```suggestion
Name: New Jersey Statewide Digital Aerial Imagery Catalog 

```

File: datasets/NJ_LiDAR.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ManagedBy:  The New Jersey Office of GIS, NJ Office of Information Technology
+UpdateFrequency: None
+Tags:
+  - lidar
+  - elevation
+  - geospatial
+  - mapping
+License:  The State of New Jersey provides the service "as is". The State makes no guarantee or warranty concerning the accuracy of information contained in the data. Also, the State makes no warranty, either expressed or implied, regarding the condition of the product or its fitness for any particular purpose. The burden for determining fitness for use lies entirely with the user. Although these data files and service have been processed successfully on computers at the State, no warranty is made by the State regarding the use of the data or service on any other system, nor does the fact of distribution constitute or imply such a warranty.
+Resources:
+  - Description: https://njgin.nj.gov/njgin/edata/elevation
```
Comment (jflasher) [inline datasets/NJ_LiDAR.yaml:19]: Please change to a text description of the bucket contents. Can add link if needed, but should also be a brief description here.

File: datasets/NJ_LiDAR.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    ARN: arn:aws:s3:::njogis-elevation
+    Region: us-west
```
Comment (jflasher) [inline datasets/NJ_LiDAR.yaml:21]: I believe this should be us-west-2.

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 21c9142436282768ef537fab11a3d40f1d454ffb
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=b4cabe0d-2752-46a5-b0a7-e45abd69e0cc%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 1c7326feda894e518c128aa52e89217d07b3949f
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=6a286f07-47bd-495a-90d3-d4172135aca0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 15ff1d118794a21dd33fd10bdc2b13db88e15327
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e3f4d180-e2f6-4e82-8026-4f2272709685%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 103378bb6c14e590dddb560ba5fee90c6956865a
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=9dd703fc-1dbb-473c-90ea-4b764301b1d0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/NJ_Imagery.yaml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  - imaging
+  - mapping
+License:  The State of New Jersey provides the service "as is". The State makes no guarantee or warranty concerning the accuracy of information contained in the data. Also, the State makes no warranty, either expressed or implied, regarding the condition of the product or its fitness for any particular purpose. The burden for determining fitness for use lies entirely with the user. Although these data files and service have been processed successfully on computers at the State, no warranty is made by the State regarding the use of the data or service on any other system, nor does the fact of distribution constitute or imply such a warranty.
+Resources:
+  - Description: New Jersey digital orthophotography archive.  
+    ARN: arn:aws:s3:::njogis-imagery
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+	Tutorials:
```
Comment (jflasher) [inline datasets/NJ_Imagery.yaml:19]: @brianembley looks like you've got some tabs in here which the linter does not like. If you could change these tabs to spaces, that should get past this error. It's like this in the other file as well. 

Also, please make sure to add a Documentation field for each entry as that's required as well. Thanks!

Comment (jflasher) [reply datasets/NJ_Imagery.yaml:19]: @brianembley just wanted to ping here again in case you wanted to get this in before out next announcement.

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: e28e5b61921a7d0fec443d5e760eae528354b3e6
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=2163eeb9-3c5b-4a34-b7a8-a3c5f00a672d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: ef986bfc3d3d22634fc2a1c9269adbebd1db5bb5
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=15f2b160-b88c-4e7d-b035-ffaaccff7a56%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jflasher** [comment]
Handled in https://github.com/awslabs/open-data-registry/pull/605

**Pull request closed without merging**